apps/public-docsite-resources/config/api-docs.js

```diff
@@ -108,6 +108,7 @@ module.exports = {
       'MultiStackedBarChart',
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv8 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33407&commitId=48fe3611a41390aa18c45d40c8ea1abe1f8d8ff8&env=prod&clientType=FLUENTUI)


  <details><summary><b>Callout</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Callout.Left bottom edge.chromium.png|3022| Changed|
|Callout.Right top edge.chromium.png|1127| Changed|

</details>


  <details><summary><b>Coachmark</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Coachmark.Collapsed.chromium.png|159| Changed|

</details>


  <details><summary><b>react-charting-VerticalBarChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-VerticalBarChart.Basic - Secondary Y Axis.chromium.png|4| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>